modules/javafx.controls/src/main/java/javafx/scene/control/Tooltip.java

```diff
@@ -859,6 +859,7 @@ public boolean isSettable(Tooltip.CSSBridge cssBridge) {
 
         private boolean hideOnExit;
         private boolean cssForced = false;
+        private boolean mouseInsideTooltip = true;
```

Comment:
Default value of this member should be false.
This boolean indicates a state where mouse is inside a Tooltip - it cannot be assumed true to begin with.